README.md

```diff
@@ -1,4 +1,8 @@
-# [WIP] Asgardeo React Native OIDC SDK & Samples
+# [DEPRECATED] Asgardeo React Native OIDC SDK & Samples
+
+## :warning: Warning!
+### React Native SDK is no longer encoraged and enriched by Asgardeo and may not work with the latest versions.
```

Comment:
The term encoraged should be changed as encouraged

Reply:
Fixed with 828fd6c8142b431d6eb09dda0135d2a55976055e